cmd/ethrex_l2/src/commands/stack.rs

```diff
@@ -277,6 +294,62 @@ impl Command {
                     current_block += U256::one();
                 }
             }
+            Command::Reconstruct {
+                genesis,
+                blobs_dir,
+                store_path,
+                coinbase,
+            } => {
+                let store = Store::new_from_genesis(
+                    store_path.to_str().expect("Invalid store path"),
+                    EngineType::Libmdbx,
```

Comment:
Should this always be libmdbx? Does it make sense to make it variable?

Reply:
We discused to add an issue to address this later